pilot/pkg/networking/core/v1alpha3/gateway.go

```diff
@@ -992,3 +979,17 @@ func buildGatewayVirtualHostDomains(hostname string, port int) []string {
 	}
 	return domains
 }
+
+// Invalid cipher suites lead Envoy to NACKing. This filters the list down to just the supported set.
+func filteredCipherSuites(server *networking.Server) []string {
+	suites := server.Tls.CipherSuites
+	ret := make([]string, 0, len(suites))
+	for _, s := range suites {
+		if security.IsValidCipherSuite(s) {
+			ret = append(ret, s)
+		} else {
+			log.Debugf("ignoring unsupported cipherSuite: %q for server %s", s, server.String())
```

Comment:
nit: server.String is fairly expensive. Maybe we can wrap in `if log.DebugEnabled` so we do not pay for it when debug log is off